spec/presenters/bank_transaction_presenter_spec.rb

```diff
@@ -0,0 +1,106 @@
+require 'rails_helper'
+
+RSpec.describe BankTransactionPresenter do
+  subject(:presenter) { described_class.new(transaction, remarks) }
+
+  let(:transaction) { create :bank_transaction, :uncategorised_credit_transaction }
+  let(:remarks) { [] }
+
+  it { is_expected.to be_a BankTransactionPresenter }
+
+  # Class methods
+  describe '#headers' do
```

Comment:
Everything looking good. Just the naming of the test. class methods can be descibed with '.class_method' e.g. '.headers' and instance methods as '#headers'

Reply:
Good catch... I was shuffling tests around and tidying this got missed!